### ISSUE TITLE
[Compatibility issue] Disabling AE2's inscriber also disables presses duplication

### ISSUE THREAD

**Issue by jamesjimmy88** [AztechMC/Modern-Industrialization#986]
As per title. MI removes all the recipes from AE2's inscriber, which means that it will also remove the ones that allow using an iron block to duplicate inscriber engineering/logic/silicon/calculation presses.
Tested on Modern-Industrialization 2.2.30

**Comment by Technici4n** [comment]
MI doesn't remove any recipe from the AE2 inscriber, it has to come from a different mod or from the pack?

**Comment by jamesjimmy88** [comment]
You are completely right. For some reason there's no JEI support for the inscriber for the current version of AE2 and since I could reproduce it with just AE2 and MI I made a wrong assumption. Sorry about that.
